### PR TITLE
instructions on how to reset a moduletest site

### PR DESCRIPTION
**Pull request by ITViking**
#### What does this PR do?
It describes how to reset a moduletest site

#### Should this be tested by the reviewer and how?
Read it through. You can test it on canary

#### Any specific requests for how the PR should be reviewed?
read and test it 

#### What are the relevant tickets?
https://reload.atlassian.net/jira/software/c/projects/DDFDRIFT/boards/464?selectedIssue=DDFDRIFT-206

### CODE REVIEW

**Review by kasperg — CHANGES_REQUESTED** [review]
:+1: Nice. I have a few questions about how we do this in practice.

File: docs/runbooks/reset-moduletest.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## When to use
+
+When a moduletest site need reseting
+
+## Procedure
+
+1. Find the moduletest site of the project that need reseting in the Lagoon UI
+2. Go to tasks
+3. Select "Copy files between environments", and set the source
+  to "main" and run the task
```
[the remark this answers is not in the document]
Comment (ITViking) [reply docs/runbooks/reset-moduletest.md:12]: I've added some more information to this step

File: docs/runbooks/reset-moduletest.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+4. Select "Copy database between environments", and set the source
+  to "main" and run the task
+5. Select "Clear Drupal Caches" and run the task
```
[the remark this answers is not in the document]
Comment (ITViking) [reply docs/runbooks/reset-moduletest.md:15]: fixed

File: docs/runbooks/reset-moduletest.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ITViking) [reply docs/runbooks/reset-moduletest.md:12]: also solved by new explanation for the file step

**Review by kasperg — APPROVED** [review]

**Pull request merged**